lib/core/src/lib/styles/_reference-variables.scss

```diff
@@ -27,3 +27,7 @@ $adf-ref-height: 40px;
 $adf-ref-width: 40px;
 $adf-ref-line-height: 40px;
 $adf-ref-margin-right: 8px;
+$adf-metadata-property-panel-border-color: rgba(0, 0, 0, 0.12);
+$adf-metadata-buttons-background-color: rgba(33, 33, 33, 0.05);
+$adf-snackbar-message-background-color: #ba1b1b;
```

Comment:
Should we really introduce new styles/designs for snackbars instead of using existing snackbars used across application to avoid introducing inconsistency? @DenysVuika @MichalKinas

Reply:
I agree with Aleksander, we should use existing snackbar design, if you want to add any changes please discuss this with Shane and in this case you should change it in the snackbar component so all snackbars would be consistent

Reply:
I think we have implemented the snack bar as per the design provided by Shane.

Reply:
But you introduced the changes in only one place which will be inconsistent with others. If you want to modify the snackbar please discuss this with Shane and introduce proper changes for snackbar component

Reply:
Now I'm using the existing snackbar design.